README.md

```diff
@@ -46,18 +49,31 @@ can pipe into it or write to it and call `.end()` when you're done.
 
 ### var epub.write(*obj*, *callback*)
 
-This is the usual stream write function. The object must have the following keys:
+This is the usual stream write function.
 
-* **chapterName** - The name of the chapter in the index.
-* **content** - The HTML content of this chapter.  This will be passed
+#### Adding a chapter
+The object must have the following keys:
+* **`chapterName`** - The name of the chapter in the index.
+* **`content`** - *(optional)* The HTML content of this chapter.  This will be passed
   through `htmltidy` in order to make it valid XHTML.
-* **index** - *(optional)* Where the chapter should show up in the index. These numbers
+* **`sourceFileName`** - *(optional)* Path to the source HTML file with chapter
```

Comment:
Yeah I agree with you, it deviates too much from the rest of the module. I think it's nice to have a module that works purely with streams.